src/SASViyaApiClient.ts

```diff
@@ -36,6 +36,7 @@ import { isAuthorizeFormRequired } from './auth/isAuthorizeFormRequired'
 import { RequestClient } from './request/RequestClient'
 import { prefixMessage } from '@sasjs/utils/error'
 import * as mime from 'mime'
+import jwtDecode from 'jwt-decode'
```

Comment:
`jwt-decode` is a transitive dependency installed via `@sasjs/utils`. I think it'd be better to keep it that way and add a helper function there that returns the decoded token - that way we could also add an interface for it and make it type safe.